README.md

```diff
@@ -1,5 +1,34 @@
 # Nexus Java SDK
 
-Java SDK for working with Nexus RPC.
+[![Java Docs](https://img.shields.io/badge/Java_Docs-red)](https://javadoc.io/doc/io.nexusrpc/nexus-sdk/latest/index.html)
+[![Continuous Integration](https://github.com/nexus-rpc/sdk-java/actions/workflows/ci.yml/badge.svg)](https://github.com/nexus-rpc/sdk-java/actions/workflows/ci.yml)
+
+Package for working with the Nexus [HTTP API](https://github.com/nexus-rpc/api).
```

Comment:
I think "Java SDK for working with Nexus RPC" from before is a better general statement than this one specifically about HTTP API.